INSTALL.md

```diff
@@ -20,6 +20,19 @@ To install the most recent ponyc:
 ponyup update ponyc release
 ```
 
+Selecting the correct distro:
+
+By default, when you use ponyup on a Glibc based Linux distro, it will install a version of ponyc that was built on the most recent Ubuntu LTS release. If your distro uses the same Glibc version, then everything will work fine.
+
+If you get an error like: `ponyc: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by ponyc)` then your version of Glibc is not compatible.
+
+You can use `ponyup show ponyc` to see all available distributions. Select the one that matches your distro using the `ponyup default` command. For example:
```

Comment:
The `ponyup show` command only lists installed packages right now. We should either wait for [`find`](https://github.com/ponylang/ponyup/issues/135) to be implemented, or direct people to https://cloudsmith.io/~ponylang/repos/releases/packages/?q=x86-64-unknown-linux to look for available distributions